node/src/run_modes.rs

```diff
@@ -428,115 +655,144 @@ parm2 - msg2\n"
-            &mut initialization_holder.streams(),
+            &mut daemon_stream_holder.streams(),
         );
-        let service_mode_exit_code = subject.go(&[], &mut service_mode_holder.streams());
+        let service_mode_exit_code = subject.go(&[], &mut node_stream_holder.streams());
 
         assert_eq!(initialization_exit_code, 1);
-        assert_eq!(initialization_holder.stdout.get_string(), "");
-        assert_eq!(
-            initialization_holder.stderr.get_string(),
-            RunModes::privilege_mismatch_message(&Mode::Initialization, true)
-        );
+        assert_eq!(daemon_stream_holder.stdout.get_string(), "");
+        let mut p_m_msg_daemon = RunModes::privilege_mismatch_message(&Mode::Initialization, true);
+        p_m_msg_daemon.push_str("\n");
+        assert_eq!(daemon_stream_holder.stderr.get_string(), p_m_msg_daemon);
         assert_eq!(service_mode_exit_code, 1);
-        assert_eq!(service_mode_holder.stdout.get_string(), "");
-        assert_eq!(
-            service_mode_holder.stderr.get_string(),
-            RunModes::privilege_mismatch_message(&Mode::Service, true)
-        );
+        assert_eq!(node_stream_holder.stdout.get_string(), "");
+        let mut p_m_msg_node = RunModes::privilege_mismatch_message(&Mode::Service, true);
+        p_m_msg_node.push_str("\n");
+        assert_eq!(node_stream_holder.stderr.get_string(), p_m_msg_node);
         let params = params_arc.lock().unwrap();
         assert_eq!(*params, vec![true, true])
     }
 
     #[test]
-    fn initialization_and_service_modes_do_not_complain_without_privilege_for_help_and_version() {
-        let mut subject = RunModes::new();
-        let run_params_arc = Arc::new(Mutex::new(vec![]));
-        let runner = RunnerMock::new()
-            .run_service_params(&run_params_arc)
-            .run_service_result(Ok(0))
-            .run_service_result(Ok(0))
-            .initialization_params(&run_params_arc)
-            .initialization_result(Ok(0))
-            .initialization_result(Ok(0));
-        subject.runner = Box::new(runner);
-        let priv_params_arc = Arc::new(Mutex::new(vec![]));
-        let privilege_dropper = PrivilegeDropperMock::new()
-            .expect_privilege_params(&priv_params_arc)
-            .expect_privilege_result(false)
-            .expect_privilege_result(false)
-            .expect_privilege_result(false)
-            .expect_privilege_result(false);
-        subject.privilege_dropper = Box::new(privilege_dropper);
-        let mut initialization_h_holder = FakeStreamHolder::new();
-        let mut initialization_v_holder = FakeStreamHolder::new();
-        let mut service_mode_h_holder = FakeStreamHolder::new();
-        let mut service_mode_v_holder = FakeStreamHolder::new();
-
-        let initialization_h_exit_code = subject.go(
-            &["--initialization".to_string(), "--help".to_string()],
-            &mut initialization_h_holder.streams(),
+    fn is_help_or_version_works() {
```

Comment:
Somewhere we need to ensure that is_help_or_version returns false when appropriate.